.bumpversion.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 9.7.2
```

Comment:
this is auto-controlled by deploy scripts;

Reply:
lgtm can we remove this one though

Reply:
Yeah, I'll revert that change. Just make sure that the deploy scripts do a major version bump, since this does introduce breaking changes

Reply:
Also, I'm realizing theres a bit more that I want to rework in the playlist library to bring it more in line with the rest of the codebase, I'll ping you again when it's ready.